### DISCUSSION TITLE
Content Publishing across different social media

### DISCUSSION THREAD

**Discussion by firozemzr** [OSBDAlliance/headlesscms#9] [Ideas]
Headless CMS should be able to publish content to different social media including but not limited to facebook, instagram, twitter and youtube using platform apis. Users of the CMS should be able to just configure their app credentials to configure headless cms.

**Comment by mohammad-arif-rahim** [comment]
API can be provided, integration will be of the users.

**Reply by firozemzr** [reply]
The requirement is to make it readily available for users, so that they can just configure their access tokens from these platforms and get readily available publishing features that they can use.

And yes, you are right, there will be a unified content publishing api that users will call with the destination channel, it will pick up the correct api for destination platform and publish using the configured token.
